resources/inspec_gem.rb

```diff
@@ -0,0 +1,41 @@
+provides :inspec_gem
+resource_name :inspec_gem
+
+property :gem_name, String, name_property: true
+property :version, String
+property :source, String
+
+default_action :install
+
+action :install do
+  if !version.nil?
+    converge_by "install InSpec version #{version}" do
```

Comment:
Should we double-check the existing version? Since it tries to install it every time once a version is set.